### ISSUE TITLE
Add "Force @user to unfollow" option to the block/report/mute menu

### ISSUE THREAD

**Issue by ghost** [florence-social/mastodon-fork#127]
Currently to force the user to unfollow (or "soft block" them) you have to block and the unblock them. This would accomplish the task in a single step using the same menu.

(Credit for idea: zigg@queer.garden)

**Comment by mattieb** [comment]
I did discover this page exists (replace queer.garden with your instance): https://queer.garden/relationships?relationship=followed_by

I like the "Remove follower" verbage better than "Force to unfollow" personally

**Comment by ghost** [comment]
I agree. Looking at the menu it probably makes sense to put the option just above "Hide everything from {domain}" and use text similar to "Remove {user} as a follower".

Hopefully that is also quite straightforward to translate for all the other locales :)

**Comment by ryliejamesthomas** [comment]
It seems really strange that this isn't already an option D:

----

I think it's worth discussing (elsewhere) what following should actually mean, and if that should be the main kind of account relationship.

Personally I think that if a social network is about people instead of brands you should have to be friends/mutuals with people to see their non-public posts. And Following should just put their public posts in your feed/timeline.

**Comment by mal0ki** [comment]
You can force unfollows from the Follows and Followers? 

![image](https://user-images.githubusercontent.com/28316305/61282322-6fec7080-a7bb-11e9-94d0-66ca84e571af.png)

**Comment by mattieb** [comment]
> You can force unfollows from the Follows and Followers?

Yeah, and while that's a useful interface for pruning a follower list, I don't think many people know this exists, and I also think the option to remove a follower from their profile would be really useful for the use case where you've observed someone interacting with you who is a follower who you'd rather not be a follower any longer, and you'd like to remove them specifically.
